libsolidity/codegen/YulUtilFunctions.cpp

```diff
@@ -3057,7 +3057,7 @@ string YulUtilFunctions::readFromMemoryOrCalldata(Type const& _type, bool _fromC
 		)")
 		("functionName", functionName)
 		("fromCalldata", _fromCalldata)
-		("validate", validatorFunction(_type))
+		("validate", validatorFunction(_type, true))
```

Comment:
This is the only functional change here. Previously, this would use an invalid opcode on out of range, now it reverts.

Reply:
And we want that for memory as well? That's what this does, doesn't it? I'm just wondering, because there's no indication of that and the PR title says ``calldata`` only...

Reply:
`validate` is only used for calldata, please check the template.

Reply:
Damn it, I must have been blind - I'll stop for today, I think I have reviewed too much already today :-).